Dockerfile

```diff
@@ -26,8 +26,10 @@ USER 1001
 
 # install all gems
 COPY --chown=deploy:deploy Gemfile Gemfile.lock .ruby-version ./
-ARG BUNDLE_FLAGS="--jobs 2 --no-cache --without development test"
-RUN bundle install ${BUNDLE_FLAGS}
+ARG BUNDLE_FLAGS="--jobs 2"
+RUN bundle config set no-cache 'true'
+RUN bundle config set without 'development test'
```

Comment:
are these related to this PR?

Reply:
they just avoid a deprecation warning in the build - not strictly essential